vuepress/docs/next/tutorials/solution/strapi.md

```diff
@@ -96,43 +94,74 @@ To log in to your Entando Strapi:
 
 ## Configuration
 
-The URL to access the Strapi instance must be provided before content can be managed. Note that the configuration of content templates relies on existing Strapi content. 
+To begin creating web content using Strapi on Entando, content and content templates are the most basic building blocks you will need to create. Any instance must include one or more content templates before content can be managed through the Strapi Content and Strapi Content List widgets. The templates provide the type and attribute parameters for each content. 
 
-Perform the steps in the following sections to properly prepare your Entando Application to use your Strapi instance:
-1. Create Strapi content for use by the Strapi Content Template Widget
+Perform the steps in the following order to properly prepare your Entando Application:
+1. Create content for use by the Strapi Content Template Widget
 2. Configure the Strapi Config Widget with the URL of your Strapi instance
-3. Configure the Strapi Content Template Widget using existing Strapi content
+3. Add the Strapi Content Template Widget using existing Strapi content
 
 ### Create Content in Strapi
 
-Before you can configure the Strapi Content Template Widget, you must add content to your Strapi instance. Refer to the Strapi [Quick Start Guide](https://docs.strapi.io/developer-docs/latest/getting-started/quick-start.html) for a tutorial on content creation. Additional information is available in the [Strapi user guide](https://docs.strapi.io/user-docs/latest/getting-started/introduction.html).
+Before you can configure the Strapi Content Template Widget, you must add content to your Strapi instance. 
```

Comment:
for a concrete example we should introduce it as such, like "as an example, to..." like what the tutorial shows you, describing the end goal. otherwise it's like the following steps are necessary for any work to be done in strapi